lib/elixir/lib/protocol.ex

```diff
@@ -290,6 +290,8 @@ defmodule Protocol do
       # otherwise generate a dummy callback
       Module.spec_to_callback(__MODULE__, {name, arity}) ||
         @callback unquote(name)(unquote_splicing(type_args)) :: term
+
+      @__generated_callbacks__ {name, arity}
```

Comment:
This is already stored in `@functions`, no?

Reply:
true